### DISCUSSION TITLE
Are these zsh-autocomplete options still valid?

### DISCUSSION THREAD

**Discussion by yrammos** [marlonrichert/zsh-autocomplete#590] [Q&A] [answered]
The plugin documentation used to include the following options, which still appear in my `.zshrc`:

- `zstyle ':autocomplete:*' min-input 1`
- `zstyle ':autocomplete:*' fzf-completion yes`
- `zstyle ':autocomplete:*' recent-dirs cdr`
- `zstyle ':autocomplete:*' widget-style menu-select`

Are they still valid? Unless I'm mistaken, they've dropped out of `readme.md` and do not appear anywhere in the code.

Many thanks.

**Comment by marlonrichert** [comment, marked as answer]
`min-input` is definitely still supported. The rest not. Here's how they work now instead:
* To let Autocomplete use `cdr` for recent dirs, you don't need to do anything.
* To use `menu-select` simply bind it to Tab, like so: `bindkey '\t' menu-select`
* To use `fzf` completion, either use its own [completion script](https://github.com/junegunn/fzf/blob/master/shell/completion.zsh) or use a third-party tool, such as [`lincheney/fzf-tab-completion`](https://github.com/lincheney/fzf-tab-completion) or [`Aloxaf/fzf-tab`](https://github.com/Aloxaf/fzf-tab).
* If you want to use both, you'll need to write your own widget that somehow decides which one to call when. I decided I no longer want to maintain this functionality, since are too many different opinions on how this should work and I don't use `fzf` myself. Or perhaps one of the authors in the previous bullet point provide this functionality (no idea; I don't use their software) or are willing to add and maintain it.
